README.md

```diff
@@ -1,2 +1,63 @@
-# aws-nextflow-workflows
-AWS CloudFormation templates for deploying Nextflow infrastructure
+# Nextflow Workflows Infrastructure
+
+The AWS infrastructure for hosting a private instance of [Nextflow Tower](https://tower.nf/) and executing [Nextflow workflows](https://nextflow.io/) is defined in this repository and deployed using [CloudFormation](https://aws.amazon.com/cloudformation/) via [Sceptre](https://sceptre.cloudreach.com/).
+
+## AWS Accounts
+
+Two AWS accounts are managed by this repository, both of which were [bootstrapped](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2058878986/Bootstrapping+AWS+Project+Accounts) using [org-formation](https://github.com/org-formation/org-formation-cli). They are defined in this [organization.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/organization.yaml) file. The two accounts are:
+
+- `workflows-nextflow-prod` for production use
+- `workflows-nextflow-dev` for testing purposes
+
+### Role-Based Access
+
+Human access to these AWS accounts (programmatically or via the Management Console) is provided using IAM roles defined in [organizations-infra](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml) (search for "WorkflowsNextflow"). These roles are individually [bound](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/README.md) to user groups in JumpCloud, which are managed by Sage IT.
+
+- **Viewer:** [`ViewOnlyAccess`](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_view-only-user)
+- **Developer-IAM-Contrained:** [`PowerUserAccess`](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_developer-power-user) and contrained IAM access, namely only to resources prefixed with `dev-`
```

Comment:
resource names prefixed with `dev-`?

Reply:
@thomasyu888: Yeah, I made the suggestion [here](https://sagebionetworks.slack.com/archives/CLH1FLNK1/p1627078948024700), and Tess implemented the role for JumpCloud [here](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L437-L471). The idea is to grant limited access to IAM such that resources can be created, but important ones cannot be messed up (like the CI service user/role).

Reply:
@thomasyu888 @BrunoGrandePhD Open to other solutions for that. Instead of using that naming scheme we could also create a protected resource policy.

Reply:
I am ok with the current solution, just wanted some clarification.

Reply:
I updated the wording.